schema/Query.graphql

```diff
@@ -28,4 +28,14 @@ type Query {
     Generate challenge to authenticate with WebAuthn
     """
     generateAuthenticatorChallenge(username: String!): AuthenticationWithWebPublicKeyCredential
+
+    """
+    Fetch life document for the user.
+    """
+    getLife(id: ObjectID!): Life
+
+    """
+    Fetch list of the lives.
+    """
+    listLives: [Life]
```

Comment:
suggest that we can return this as `[Life!]!` so we don't need to handle null/undefined values.
We either care about wether it is an empty array, and the life data it contains